man/GGIR.Rd

```diff
@@ -783,6 +781,14 @@ GGIR(mode = \Sexpr{format(formals(GGIR::GGIR)[["mode"]])},
       \item{max_calendar_days}{
       Numeric (default = \Sexpr{GGIR::load_params()[["params_cleaning"]][["max_calendar_days"]]}).
       The maximum number of calendar days to include (set to zero if unknown).}
+      
+      
+      \item{nonWearEdgeCorrection}{
+      Boolean (default = \Sexpr{GGIR::load_params()[["params_cleaning"]][["nonWearEdgeCorrection"]]}).
+        If TRUE then the edges are corrected following description in vanHees2013
+        as has been the default since then.
```

Comment:
Should we add some extra information on when the user could be interested in setting this to FALSE? Lab studies and/or short recordings?

Reply:
thanks, now fixed